examples/kubernetes/servicemesh/envoy/test-application.yaml

```diff
@@ -0,0 +1,326 @@
+---
+apiVersion: v1
+data:
+  Corefile: ". {\n\t\t\t\tlocal\n\t\t\t\tready\n\t\t\t\tlog\n\t\t\t}"
```

Comment:
nit: does this need to be encoded this way? this could be a bit more readable if it was a multiline field.

Reply:
good point, I just updated with more user friendly format :100: